### PR TITLE
Refactor: abstracted out symbol used to delimit multiple arguments for prefixes

### LINKED ISSUE #178: Parsing: use alternative character as separator for multiple arguments
Currently multiple argument inputs are separated with "," to delimit the start and end of each argument for each prefix.

The issue w this occurs when arguments themselves involve the use of the "," character, hence a change must be made to abstract the character used as the separator for multiple input arguments in the event this occurs with other characters as well.

### PR DESCRIPTION
**Pull request by aidanaden**
Closes #178

Current delimiter is "  " instead of "," due to "|" and "+" both causing issues with the tests, alternative suggestions are v welcome!

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/RecipeBookSyntax.java [hunk 1]
```diff
@@ -15,4 +15,6 @@
 
     /** Used to indicate index in a command */
     public static final Prefix PREFIX_INDEX = new Prefix("-x ");
+
+    public static final String SEPARATOR_SYMBOL = "  ";
```
Comment (yusufaine) [inline src/main/java/seedu/address/logic/parser/RecipeBookSyntax.java:19]: This change should allow separating values by the pipe symbol.
```suggestion
    public static final String SEPARATOR_SYMBOL = "\\|";
```

**Review by yusufaine — APPROVED** [review]
LGTM, suggested a change that allows split by |, also need your help to change line 34 in FindCommandParser to use `SEPARATOR_SYMBOL`

File: src/main/java/seedu/address/logic/parser/RecipeBookSyntax.java (hunk 1 above)
Comment (Oddcorner) [reply src/main/java/seedu/address/logic/parser/RecipeBookSyntax.java:19]: I agree with this, looks good!

**Review by Oddcorner — APPROVED** [review]
LGTM! Nice abstraction to make modification in the future easier!

**Pull request merged**